--- base
+++ macos
@@ -33,7 +33,7 @@
 ## ALIASES
 
 # Set colored LS output
-alias ls='ls --color=auto'
+alias ls='ls -G'
 # Calendar alias
 alias cal='cal -m --week -3'
 # 'open' command
@@ -46,6 +46,13 @@
 export CLICOLOR=1
 # Set the default editor
 export EDITOR=vim
+# Set colors to GNU-like look
+export LSCOLORS=ExGxFxdxCxdxdxabagacac
 
 
 ## OTHER
+# Bash completion on macOS
+if [ -f /opt/local/etc/profile.d/bash_completion.sh ]; then
+    . /opt/local/etc/profile.d/bash_completion.sh
+fi
+
